donate/templates/pages/core/ways_to_give_page.html

```diff
@@ -0,0 +1,94 @@
+{% extends "pages/base_page.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Ways to Give" %}{% endblock %}
+
+{% block template_name %}app--ways-to-give{% endblock %}
+
+{% block header %}
+<div class="app__header">
+    <h1>{% trans "Ways to Give" %}</h1>
+    <div><a href="{{ways_to_give_link}}" class="button button--outline button--dark-bg">{% trans "donate now" %}</a></div>
+</div>
+{% endblock %}
+
+{% block navigation %}{% endblock %}
+
+{% block content %}
+<div class="container">
+    <h2>{% trans "Credit / Debit and PayPal" %}</h2>
+    <p>
+        {% blocktrans trimmed %}
+        <a href="{{ways_to_give_link}}">Go to this link</a> to make a secure online donation right now.
+        {% endblocktrans %}
+    </p>
+
+    <h2>{% trans "Bank Transfer / SEPA" %}</h2>
+    <p><strong>{% trans "New!" %}</strong> {% trans "We are now able to accept donations in Euros through Single European Payment Area (SEPA) transfer, and British Pounds through BACS transfer." %}</p>
+    <p>{% trans "Please remember that donations are deductible for U.S. income tax purposes only." %}</p>
+    <p>
+        {% blocktrans with address="331 E. Evelyn Ave, Mountain View, CA 94041 USA" trimmed %}
+        If your bank requests our physical address, it is {{address}}.
+        {% endblocktrans %}
+    </p>
+
+    <h3>{% trans "Euros:" %}</h3>
+    <p>
+        <b>Account Currency:</b> Euro (EUR)<br>
+        <b>Receiving Bank (Pay to):</b> Standard Chartered Bank<br>
+        <b>City & Country/Region:</b> Frankfurt, Germany<br>
+        <b>SWIFT/BIC Code:</b> SCBLDEFX<br>
+        <b>Beneficiary Name (For credit to):</b> SVB-Mozilla Foundation<br>
+        <b>IBAN:</b> DE67512305000500136802
+    </p>
+
+    <h3>{% trans "British Pound Sterling:" %}</h3>
+    <p>
+        <b>Account Currency:</b> British Pound Sterling (GBP)<br>
+        <b>Receiving Bank (Pay to):</b> National Westminster Bank<br>
+        <b>City & Country/Region:</b> London, United Kingdom<br>
+        <b>Bank Sort Code:</b> 60-00-04<br>
+        <b>Account Number (8 digits):</b> 10017496<br>
+        <b>Beneficiary Name (For credit to):</b> SVB RE Mozilla Foundation
+    </p>
+    <p>
+        <b>SWIFT/BIC Code:</b> NWBKGB2L<br>
+        <b>IBAN:</b> GB77NWBK60000410017496
+    </p>
+
+    <h2>{% trans "Check (via postal service)" %}</h2>
+    <p>
+        {% blocktrans trimmed %}
+        Please send a check <strong>including your email address on the memo line</strong>, and payable (payee) to “Mozilla Foundation”. Please note that the Mozilla Foundation can only accept checks in US dollars drawn on a US bank. Please do <strong>not</strong> send cash.
```

Comment:
We should probably move the payee info to a variable too. In the old website I only added a localization note asking not to translate, but this is safer

```suggestion
        Please send a check <strong>including your email address on the memo line</strong>, and payable (payee) to “{{payee}}”. Please note that the Mozilla Foundation can only accept checks in US dollars drawn on a US bank. Please do <strong>not</strong> send cash.
```